styleguide/source/_patterns/01-molecules/06-components/20-testimonial/testimonial.twig

```diff
@@ -1,12 +1,12 @@
-<article class="testimonial">
+<div class="testimonial">
   <div class="testimonial_image">
       <span style="background-image: url(https://ipsumimage.appspot.com/600x400?l=3:2|600x400&s=36)"></span>
   </div>
   <div class="testimonial_text">
-      {% include 'atoms-paragraph' with { 'class': 'testimonial_quote', 'content': '"Testimonial ipsum dolor sit amet, fringilla leo libero integer ut, elementum maecenas mi amet commodo pellentesque, quis odio. Dolor posuere tempor quam et, maecenas auctor hac urna ac eu, sed in."' } %}
+      {% include 'atoms-paragraph' with { 'class': 'testimonial_quote', 'content': testimonial.quote } %}
       <p class="testimonial_speaker">
-          <span class="testimonial_speaker_person">—Dr. Bob Jones</span>,
-          <span class="testimonial_speaker_title">Medicology</span>
+          <span class="testimonial_speaker_person">{{ testimonial.speakerName ? testimonial.speakerName : 'Bob M. Doctor' }}</span>,
```

Comment:
addressed this in [this commit](https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/157/commits/45352a2216715cf97527f1e7414440a8b93e9d2d)